src/node.cc

```diff
@@ -4234,6 +4234,11 @@ inline int Start(Isolate* isolate, IsolateData* isolate_data,
   Environment env(isolate_data, context, v8_platform.GetTracingAgent());
   env.Start(argc, argv, exec_argc, exec_argv, v8_is_profiling);
 
+  TRACE_EVENT_METADATA1("__metadata", "version", "node", NODE_VERSION + 1);
```

Comment:
nit: I'm trying to understand the `+ 1`. If you're skipping the `v` in the version string: 1) why? 2) it would be good to add a comment explaining what the `+ 1` does as it is not immediately obvious. I would prefer if the `v` was included too.

Reply:
i thought we had a lint rule against messing with pointers like that

Reply:
It is for consistency with the value of `process.versions.node` (which does not include the `v`)